src/Tools/dotnet-monitor/Egress/EgressProvider.cs

```diff
@@ -44,47 +44,6 @@ protected EgressProvider(ILogger logger)
             Logger = logger;
         }
 
-        /// <summary>
-        /// Egress a stream via a callback by returning the stream from the callback.
-        /// </summary>
-        /// <param name="options">Described to where stream data should be egressed.</param>
-        /// <param name="action">Callback that is invoked in order to get the stream to be egressed.</param>
-        /// <param name="artifactSettings">Describes data about the artifact, such as file name and content type.</param>
-        /// <param name="token">The token to monitor for cancellation requests.</param>
-        /// <returns>A task that completes with a value of the identifier of the egress result. Typically,
-        /// this is a path to access the stream without any information indicating whether any particular
-        /// user has access to it (e.g. no file system permissions or SAS tokens).</returns>
-        public virtual Task<string> EgressAsync(
-            string providerType,
-            string providerName,
-            TOptions options,
-            Func<CancellationToken, Task<Stream>> action,
-            EgressArtifactSettings artifactSettings,
-            CancellationToken token)
-        {
-            Func<Stream, CancellationToken, Task> wrappingAction = async (targetStream, token) =>
-            {
-                using var sourceStream = await action(token);
-
-                int copyBufferSize = options.CopyBufferSize.GetValueOrDefault(0x100000);
```

Comment:
I noticed that `CopyBufferSize` was being used here, but then realized that all the deleted code appears to be an unused code path. If there's a reason to leave this in I can revert it, but I believe it's just stale code.